app/Web/Mail.php

```diff
@@ -71,7 +71,8 @@ public function to(string $mail)
      */
     public function subject(string $text)
     {
-        $this->subject = htmlentities($text);
+        $this->subject = utf8_decode($text);
+        $this->subject = mb_encode_mimeheader($text);
```

Comment:
I would not like add the extension mbstring for only an encoding, I think it's better looking for another encondig, like base64 or similars.